src/main/java/gregtech/loaders/recipe/chemistry/SeparationRecipes.java

```diff
@@ -566,4 +569,26 @@ public static void init() {
                 .duration(300).EUt(2).buildAndRegister();
 
     }
+
+    @SuppressWarnings("unchecked")
+    private static List<Tuple<ItemStack, Integer>> getGrassSeedEntries() {
```

Comment:
Tuple can be replaced with `Object2IntMap.Entry<ItemStack>`. Instances of this type can be created with `new AbstractObject2IntMap.BasicEntry<>(object, int)`